src/app/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import CalendarIcon from '../assets/CalendarIcon';
+import OverviewIcon from '../assets/OverviewIcon';
+import styles from './Navigation.module.css';
+import { Link } from 'react-router-dom';
+
+export type NavigationProps = {
+  activeLink: 'calendar' | 'overview';
+  className?: string;
+  to: string;
+};
+
+function Navigation({ activeLink, className }: NavigationProps): JSX.Element {
```

Comment:
```suggestion
function Navigation({ activeLink, className, to }: NavigationProps): JSX.Element {
```

Reply:
I don't agree the `to` prop wouldn't make sense, since there are several links in this component